repo/packages/H/hdfs/5/marathon.json.mustache

```diff
@@ -0,0 +1,117 @@
+{
+  "id": "/{{service.name}}",
+  "cmd": "./scheduler/bin/hdfs-scheduler server ./scheduler/conf/scheduler.yml",
+  "instances": 1,
+  "cpus": {{service.cpus}},
+  "mem": {{service.mem}},
+  "requirePorts": true,
+  "portDefinitions": [
+    {
+      "port": {{service.api_port}},
+      "protocol": "tcp",
+      "name": "api",
+      "labels": {}
+    }
+  ],
+"uris": [
+  "{{resource.assets.uris.scheduler-zip}}",
+  "{{resource.assets.uris.jre-tar-gz}}"
+],
+"healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": { "value": "curl -f -X GET http://$HOST:{{service.api_port}}/admin/healthcheck" },
+      "gracePeriodSeconds": 300,
+      "intervalSeconds": 60,
+      "timeoutSeconds": 60,
+      "maxConsecutiveFailures": 3
+    }
+],
+"readinessChecks": [
+    {
+      "name": "hdfsUpdateProgress",
+      "protocol": "HTTP",
+      "path": "/v1/plan",
+      "portName": "api",
+      "interval": 10000,
+      "timeout": 10000,
+      "httpStatusCodesForReady": [200],
+      "preserveLastResponse": true
+    }
+],
+"labels": {
+  "DCOS_PACKAGE_FRAMEWORK_NAME": "{{service.name}}",
+  "DCOS_MIGRATION_API_VERSION": "v1",
+  "DCOS_MIGRATION_API_PATH": "/v1/plan",
+  "MARATHON_SINGLE_INSTANCE_APP":"true"
+},
+"upgradeStrategy":{
+  "minimumHealthCapacity": 0,
+  "maximumOverCapacity": 0
+},
+{{#service.secret_name}}
+"secrets": {
+  "serviceCredential": {
+    "source": "{{service.secret_name}}"
+  }
+},
+{{/service.secret_name}}
+"env":{
+{{#service.secret_name}}
+"DCOS_SERVICE_ACCOUNT_CREDENTIAL": { "secret": "serviceCredential" },
+"MESOS_MODULES": "file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
+"MESOS_AUTHENTICATEE": "com_mesosphere_dcos_ClassicRPCAuthenticatee",
+"SSL_CERT_FILE": "/run/dcos/pki/tls/certs/scheduler.crt",
+"SSL_KEY_FILE": "/run/dcos/pki/tls/private/scheduler.key",
+"SSL_CA_FILE": "/run/dcos/pki/CA/certs/ca.crt",
+"SSL_ENABLED": "true",
+{{/service.secret_name}}
```

Comment:
So you're still bundling scheduler SSL in with authentication, which is fine for Argus, but for other customers, we may want to be able to enable SSL without authentication or v.v. See the Spark PR https://github.com/mesosphere/universe/pull/524